packages/fxa-payments-server/src/components/PlanErrorDialog/index.stories.tsx

```diff
@@ -15,13 +20,18 @@ const plans: FetchState<Plan[], any> = {
   result: PLANS,
 };
 
-storiesOf('components/PlanErrorDialog', module)
-  .add('no plan for product', () => (
+const storyWithContext = (plans: any, storyName?: string) => {
```

Comment:
Would this be `plans: FetchState<Plan[], any>`, instead of `plans: any`?

Reply:
Updated!